zerver/webhooks/uptimerobot/doc.md

```diff
@@ -0,0 +1,36 @@
+Receive Zulip notifications for your Uptime Robot monitors!
```

Comment:
Maybe "... notifications from Uptime Robot"?  That reads more cleanly.

Reply:
Changed.